src/components/Accordion.js

```diff
@@ -24,6 +24,38 @@ document.addEventListener(
   false
 );
 
+document.addEventListener(
+  "focusin",
+  onAccordionFocus => {
+    const { target } = onAccordionFocus;
+    changeFocusClass(target, "open");
+  },
+  false
+);
+
+document.addEventListener(
+  "focusout",
+  onAccordionFocus => {
```

Comment:
One other naming thing/pattern here, I propose any type of listener follows this pattern.

```js
document.addEventListener(
  "focusout",
   handleDocumentClick,
   false
);
```

```js
const handleDocumentClick = clickEvent => {
  const { target } = clickEvent;
};
```

There are two parts here, any event listener function will have function associated with it that starts with `handle` and the parameter passed to that function will contain `event` in the name.

There is nothing in the airbnb style guide that covers this. This pattern has stemmed from the react [docs](https://reactjs.org/docs/handling-events.html#passing-arguments-to-event-handlers), and other react projects.

@sgrg1 @tmccoy529 @theilman7

Reply:
@baltimorecounty/front-end-developers guys, this naming stuff might be slightly cumbersome to start, but should become much easier as we go along.

Reply:
@martypowell @theilman7 @tmccoy529 . I am ok with it since we are already using it with modal and it make sense.

Reply:
This seems helpful as it connects the  items together in their name, which again could make it easier to step in to and troubleshoot.